### ISSUE TITLE
Aria:TabPanel 

### ISSUE THREAD

**Issue by tonyhallett** [mdn/content#15515]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tabpanel_role

### What specific section or headline is this issue about?

Description

### What information was incorrect, unhelpful, or incomplete?

> Only one tabpanel is visible at a time.

This document does allow for a multi-selectable tab list

### What did you expect to see?

That agrees with the linked WAI-ARIA

### Do you have any supporting links, references, or citations?

https://w3c.github.io/aria/#tablist

### Do you have anything more you want to share?

The example code pen does not follow the https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tablist_role#keyboard_interactions
The left and right arrows does not cycle.  Unlike https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html

Bad links
https://w3c.github.io/aria-practices/examples/tabs/tabs-1/tabs.html
should be https://w3c.github.io/aria-practices/examples/tabs/tabs-automatic.html

https://w3c.github.io/aria-practices/examples/tabs/tabs-2/tabs.html
should be https://w3c.github.io/aria-practices/examples/tabs/tabs-manual.html

Also I could not see down arrow focus mentioned by the W3
> If the tablist precedes the tabpanel in source order, the user will expect to be able to step through the remaining tabs in focus order or select the down arrow to give focus to the visible tabpanel when navigating by keyboard.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/accessibility/aria/roles/tabpanel_role`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tabpanel_role
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/roles/tabpanel_role/index.md
* Last commit: https://github.com/mdn/content/commit/43b0e8529065cc4f0f22497d1d2eadecd3d77dbc
* Document last modified: 2022-04-02T04:30:46.000Z

</details>
